Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -21,13 +21,15 @@ import SourceKitD
 /// request, such as the necessary macro expansion edits.
 struct MacroExpansion: RefactoringResponse {
   /// The title of the refactoring action.
-  var title: String
+  let title: String
 
   /// The URI of the file where the macro is used
-  var uri: DocumentURI
+  let uri: DocumentURI
 
   /// The resulting array of `RefactoringEdit` of a semantic refactoring request
-  var edits: [RefactoringEdit]
+  let edits: [RefactoringEdit]
+
+  var expansionURIs: [DocumentURI]?
```

Comment:
Could you add a doc comment?

Also, out of curiosity: Why did you change all properties to `let` and kept this one as a `var`?

Reply:
For structs I often prefer `var`. Structs are value types so shared mutable state is not really an issue and if the properties are `let`, you can’t create a modified copy as follows.

```swift
var myCopy = someStruct
myCopy.someProperty = 2
doSomething(with: myCopy)
```

Reply:
That makes sense now.